### PR TITLE
Add guide to Exact MAU Tables

### PR DESCRIPTION
**Pull request by jklukas**
The conceptual model text here is adapted from a [Smoot requirements doc](https://docs.google.com/document/d/1L8tWDUjccutGGAldhpypRtPCaw3kkXboPUTtTZb02OA/edit) authored by @jmccrosky.

Publishing a draft here early to raise awareness and start gathering feedback.

cc @relud

### CODE REVIEW

File: src/datasets/bigquery/exact_mau/reference.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
+# Exact MAU Data
+
+## Introduction
+
+This article introduces the usage of and methodology behing the "exact MAU"
```
Comment (relud) [inline src/datasets/bigquery/exact_mau/reference.md]:
```suggestion
This article introduces the usage of and methodology behind the "exact MAU"
```

File: src/datasets/bigquery/exact_mau/reference.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+A simple _usage criterion_ is "All Desktop Activity", which includes all Firefox Desktop users that we have any data (telemetry ping) for on the day in question.  The simplest slice is "All" which places no restrictions on the sub-population.
+
+For example, the metric "Daily Active Users (DAU)" with usage criteria "All Desktop Activity" and slice "All" involves summarizing data on all users of Firefox Desktop over a single day.
+
+## Usage Criteria
+
+A _usage criterion_ is any product or feature for which we wish to calculate
+metrics.  It may be something simple like "All Desktop Activity" (as above) or, 
+similarly, "All Mobile Activity".  It may also be something more specific like
+"Desktop 5 URI Active" corresponding to calculation of
```
Comment (relud) [inline src/datasets/bigquery/exact_mau/reference.md]: `"All Mobile Activity"` and `"All Desktop Activity"` sound like natural language, but this one doesn't. How about this?
```suggestion
"Desktop Total Daily URI >= 5" corresponding to calculation of 
```

File: src/datasets/bigquery/exact_mau/reference.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+- for all users (whatever notion of user makes sense for the data, generally profiles) in a particular sub-population
+- where the sub-population will include users that meet the specified usage criteria and are in the specified slice.
+
+A simple _usage criterion_ is "All Desktop Activity", which includes all Firefox Desktop users that we have any data (telemetry ping) for on the day in question.  The simplest slice is "All" which places no restrictions on the sub-population.
+
+For example, the metric "Daily Active Users (DAU)" with usage criteria "All Desktop Activity" and slice "All" involves summarizing data on all users of Firefox Desktop over a single day.
+
+## Usage Criteria
+
+A _usage criterion_ is any product or feature for which we wish to calculate
```
Comment (relud) [inline src/datasets/bigquery/exact_mau/reference.md]: This definition seems really vague. If _slice_ maps to `GROUP BY` does _usage criteria_ map to `WHERE`? Would calling this a condition make sense?

File: src/datasets/bigquery/exact_mau/reference.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    country = 'US'
+    AND campaign = 'whatsnew'
+GROUP BY
+    submission_date
+ORDER BY
+    submission_date
+```
+
+Additional usage criteria may be added in the future as new columns named
+`mau_*`, etc. where the suffix describes the usage criterion.
```
Comment (relud) [inline src/datasets/bigquery/exact_mau/reference.md]: right now we are using the prefix to describe the criterion

File: src/datasets/bigquery/exact_mau/reference.md (hunk 2 above)
Comment (relud) [reply src/datasets/bigquery/exact_mau/reference.md]: Or to be consistent with the column naming:
```suggestion
"Desktop Visited 5 URI" corresponding to calculation of 
```

Comment (relud) [reply src/datasets/bigquery/exact_mau/reference.md]: Or to use the terminology in active_dau.md
```suggestion
"Active Users Desktop Activity" corresponding to calculation of 
```

File: src/datasets/bigquery/exact_mau/reference.md (hunk 4 above)
Comment (jklukas) [reply src/datasets/bigquery/exact_mau/reference.md]: Yes, I intended to change this. Thanks for catching.

File: src/datasets/bigquery/exact_mau/reference.md (hunk 3 above)
Comment (jklukas) [reply src/datasets/bigquery/exact_mau/reference.md]: cc @jmccrosky as this terminology is his work. I'm going to maintain "usage criterion" for the time being to match Jesse's smoot work.

**Comment by jklukas** [comment]
@relud - See https://github.com/mozilla/bigquery-etl/pull/55 for changes to address naming consistency that you brought up.

I've updated the text in this PR to match those changes, so will hold off on merging this until the ETL changes are merged and deployed.

File: src/datasets/bigquery/exact_mau/reference.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+
+A simple _usage criterion_ is "All Desktop Activity", which includes all Firefox Desktop users that we have any data (telemetry ping) for on the day in question.  The simplest slice is "All" which places no restrictions on the sub-population.
+
+For example, the metric "Daily Active Users (DAU)" with usage criteria "All Desktop Activity" and slice "All" involves summarizing data on all users of Firefox Desktop over a single day.
+
+## Usage Criteria
+
+Active user counts must always be calculated in reference to some specific
+_usage criterion_, a binary condition we use to determine whether a given
+user should be considered "active" in a given product or feature.
```
Comment (relud) [inline src/datasets/bigquery/exact_mau/reference.md]: ❤️ this is much more clear, I really like it

**Pull request merged**